Dockerfile

```diff
@@ -44,6 +47,13 @@ RUN  npm install --unsafe-perm \
   && for plugin in plugins/enabled/*; do cd "$plugin"; npm install --unsafe-perm; cd /var/app; done \
   && for plugin in plugins/available/*; do cd "$plugin"; npm install --unsafe-perm; cd /var/app; done
 
+ADD ./bin /var/app/bin
+ADD ./config /var/app/config
+ADD ./docker-compose /var/app/docker-compose
+ADD ./lib /var/app/lib
+ADD ./protocols /var/app/protocols
+ADD ./default.config.js /var/app/default.config.js
+
```

Comment:
I don't get why you change those `ADD` lines. It will result in a breaking change on `kuzzlerc` possible location https://github.com/dominictarr/rc/blob/master/README.md#standards .

Reply:
IMO, you should revert changes from this file

Reply:
Yes, it's a mistake